contracts/optimistic-ethereum/iOVM/bridge/tokens/iOVM_L2DepositedERC20.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0;
+pragma experimental ABIEncoderV2;
+
+/* Interface Imports */
+import { IUniswapV2ERC20 } from "../../../libraries/standards/IUniswapV2ERC20.sol";
+
+/**
+ * @title iOVM_L2DepositedERC20
+ */
+interface iOVM_L2DepositedERC20 is IUniswapV2ERC20 {
+
+    /**********
+     * Events *
+     **********/
+    event WithdrawalInitiated(address indexed _from, address _to, uint256 _amount);
+    event DepositFinalized(address indexed _to, uint256 _amount);    
+
+    /********************
+     * Public Functions *
+     ********************/
+    function withdraw(uint _amount) external;
+    function withdrawTo(address _to, uint _amount) external;
+    
+    /*************************
+     * Cross-chain Functions *
+     *************************/
+    function finalizeDeposit(address _to, uint _amount) external;
+}
```

Comment:
Note that a lot of these interfaces don't use the standard function naming style, but imo it's probably fine to keep it this way for the interface contracts as long as the function name doesn't get too long